pandas-stubs/core/series.pyi

```diff
@@ -3983,25 +4488,34 @@ class Series(IndexOpsMixin[S1], NDFrame):
         level: None = ...,
         numeric_only: _bool = False,
         **kwargs: Any,
+    ) -> Any: ...
```

Comment:
same comment as for `mean` and `std`.  Most people will want the result to be `float`.  Otherwise they have to `cast` to `Series[float]` to get the right type of the result (or cast the result)